recurrence/base.py

```diff
@@ -896,36 +901,18 @@ def serialize_rule(rule):
     items = []
 
     if obj.dtstart:
-        if obj.dtstart.tzinfo:
-            dtstart = serialize_dt(obj.dtstart.astimezone(pytz.utc))
-        else:
-            dtstart = serialize_dt(
-                localtz().localize(obj.dtstart).astimezone(pytz.utc))
-        items.append((u'DTSTART', dtstart))
+        items.append((u'DTSTART', serialize_dt(obj.dtstart)))
```

Comment:
These methods previously used localize + UTC conversion which is handled internally by `serialize_dt` and `to_utc` so the code paths are considerably simplified.